<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwHorloge</class>
 <widget class="QWidget" name="qwHorloge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>101</height>
   </rect>
  </property>
  <widget class="QLabel" name="lblFondHM">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG14 Classic</family>
     <pointsize>72</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(24, 40, 24);
background-color: None;</string>
   </property>
   <property name="text">
    <string>~~:~~</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblHM">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG14 Classic</family>
     <pointsize>72</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(127, 255, 127);
background-color: None;</string>
   </property>
   <property name="text">
    <string>23:59</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblFondS">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>47</y>
     <width>81</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG14 Classic</family>
     <pointsize>36</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(24, 40, 24);
background-color: None;</string>
   </property>
   <property name="text">
    <string>~~</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblS">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>47</y>
     <width>81</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DSEG14 Classic Mini</family>
     <pointsize>36</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(127, 255, 127);
background-color: None;</string>
   </property>
   <property name="text">
    <string>59</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
